easybuild/easyconfigs/n/NewHybrids/NewHybrids-1.1_Beta3-GCC-10.2.0.eb

```diff
@@ -0,0 +1,42 @@
+# Author: Denis Krišťák (INUITS)
+# License: GPLv2
+
+easyblock = 'Binary'
+
+name = 'NewHybrids'
+version = '1.1_Beta3'
+
+homepage = 'https://ib.berkeley.edu/labs/slatkin/eriq/software/software.htm#NewHybs'
+description = """This implements a Gibbs sampler to estimate the posterior probability that genetically sampled
+individuals fall into each of a set of user-defined hybrid categories."""
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+extract_sources = True
+
+sources = [{
+    'filename': SOURCE_TAR_GZ,
+    'git_config': {
+        'url': 'https://github.com/eriqande',
+        'repo_name': 'newhybrids',
+        'commit': '6fc8fd9ca6e64ee0be08af12ec8bfe73a7de00cc',
+        'recursive': True,
+    }
+}]
+checksums = ['2cc1d3e95ce167312f3be971a3acd1951bf1cdd49313696a86ee22b7685fc56c']
```

Comment:
@deniskristak You should use `checksums = [None]` here, because when using `git_config` the source tarball is created locally, and won't be identical on every system.

The commit used here serves as checksum..

Reply:
aah, okay, interesting